pkg/api/v1/routes/handlers.go

```diff
@@ -163,8 +163,71 @@ func (r *Routes) serverConditionCreate(c *gin.Context) (int, *v1types.ServerResp
 		}
 	}
 
+	return r.conditionCreate(otelCtx, newCondition, serverID, facilityCode)
+}
+
+func (r *Routes) serverEnroll(c *gin.Context) (int, *v1types.ServerResponse) {
+	otelCtx, span := otel.Tracer(pkgName).Start(c.Request.Context(), "Routes.serverEnroll")
+	span.SetAttributes(
+		attribute.KeyValue{Key: "serverId", Value: attribute.StringValue(c.Param("id"))},
+		attribute.KeyValue{Key: "IP", Value: attribute.StringValue(c.Param("ip"))},
+		attribute.KeyValue{Key: "user", Value: attribute.StringValue(c.Param("user"))})
+	defer span.End()
+
+	repo, ok := r.repository.(*store.Serverservice)
+	if !ok {
+		return http.StatusInternalServerError, &v1types.ServerResponse{
+			Message: "add-server only support server-service store",
+		}
+	}
+
+	serverID, err := uuid.Parse(c.Param("uuid"))
+	if err != nil {
+		r.logger.WithFields(logrus.Fields{
+			"serverID": c.Param("uuid"),
+		}).Info("bad serverID")
+
+		return http.StatusBadRequest, &v1types.ServerResponse{
+			Message: err.Error(),
+		}
+	}
+
+	facilityCode := c.Param("facility")
+	// Creates a server record in FleetDB.
+	if err = store.AddServer(c.Request.Context(), repo, serverID, c.Param("facility"), c.Param("ip"), c.Param("user"), c.Param("pwd")); err != nil {
```

Comment:
We should pull out/sanity check our parameters up front in the function. We keep calling `c.Param(token)` and that's annoying.